l1-contracts/src/core/libraries/DataStructures.sol

```diff
@@ -94,4 +94,10 @@ library DataStructures {
     uint256 blockNumber;
   }
   // docs:end:registry_snapshot
+
+  struct OutboxMessageMetadata {
```

Comment:
Is this used in non-tests? If it is just a test thing it should not be in here. Notice here that the portals are not part of the core, they are external.